### PR TITLE
The homework for Lesson 8 was done // Mariia Olivenbaum

### PR DESCRIPTION
**Pull request by maria-olivenbaum**

### CODE REVIEW

File: MariaOlivenbaum/HW8/Task 3/user_input.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import area_claculation as area
```
Comment (kolyasalubov) [inline MariaOlivenbaum/HW8/Task 3/user_input.py:1]: good job

File: MariaOlivenbaum/HW8/Task 1/main.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from models import *
```
Comment (kolyasalubov) [inline MariaOlivenbaum/HW8/Task 1/main.py:1]: good job

File: MariaOlivenbaum/HW8/Task 2/main.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import password_validation as pass_valid
```
Comment (kolyasalubov) [inline MariaOlivenbaum/HW8/Task 2/main.py:1]: good job

File: MariaOlivenbaum/HW8/Task 2/password_validation.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+def validation (password):
+    flag_upper_letter = False
+    flag_low_letter = False
+    flag_number = False
+    flag_symbol = False
+
```
Comment (kolyasalubov) [inline MariaOlivenbaum/HW8/Task 2/password_validation.py:6]: good idea
but look at this implementation:
import re

password = input("Enter your password:\n")

if re.match("^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[$#@])(?=.{6,16}$).*", password):
    print("Nice pass! Psss, I won't tell anyone ;)")
else:
    print("Password is not valid. Check if you have:"
          """
    At least 1 letter between[a-z] and 1 letter between[A-Z].
    At least 1 number between[0-9].
    At least 1 character from [$#@].
    Minimum length 6 characters.
    Maximum length 16 characters.
          """
           )

File: MariaOlivenbaum/HW8/Task 3/area_claculation.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import math
```
Comment (kolyasalubov) [inline MariaOlivenbaum/HW8/Task 3/area_claculation.py:1]: good job

**Review by kolyasalubov — APPROVED** [review]

File: MariaOlivenbaum/HW8/Task 2/password_validation.py (hunk 4 above)
Comment (maria-olivenbaum) [reply MariaOlivenbaum/HW8/Task 2/password_validation.py:6]: о, я зрозуміла, дякую). бо я скіпнула 9те заняття  через роботу. тільки передивилася, що там і re.* обговорювали, і множини 😅

**Pull request merged**